net/interfaces/interfaces_linux.go

```diff
@@ -135,14 +135,16 @@ func DefaultRouteInterface() (string, error) {
 }
 
 var zeroRouteBytes = []byte("00000000")
+var procNetRoutePath = string("/proc/net/route")
```

Comment:
the `string` part is redundant. and you could also make it a const instead of a var.

Reply:
Done. It needs to be var because a unit test changes it to point to a tmpfile.